features/steps/step_pingbot.py

```diff
@@ -20,10 +20,12 @@ def step_impl(context):
 
 @when('we send message')
 def step_impl(context):
-    context.lobby_page.room_send_msg('/clear')
-    context.lobby_page.room_send_msg('/ping me with 223')
+    # context.lobby_page.room_send_msg('/clear')
+    # context.lobby_page.room_send_msg('/ping me with 223')
```

Comment:
If i understanded correctly, U commenter pingbot func for continious fixing  ? If no, I think we dont need empty funcs in master.

Reply:
Since we agreed that I will fix pingbot feature, I leave it in this step file, after this PR will merge, I will create another branch for pingbot